SOURCES/cve-2019-1547.patch

```diff
@@ -0,0 +1,228 @@
+From 21c856b75d81eff61aa63b4f036bb64a85bf6d46 Mon Sep 17 00:00:00 2001
+From: Billy Brumley <bbrumley@gmail.com>
+Date: Sat, 7 Sep 2019 10:50:58 +0300
+Subject: [PATCH] [crypto/ec] for ECC parameters with NULL or zero cofactor,
+ compute it
+
+Backport notes:
+The CHANGES file containing the changelog for each version has not
+been updated by previous backport, so its modifications were dropped
+because it wasn't applying.
+
+Original commit log:
```

Comment:
And this could be removed if the backport notes are moved before the quoted commit message.

Reply:
fixed